### ISSUE TITLE
ACC-264.1 (Level A) _ Modify Modules _ Button _ Name of dialog is not read 

### ISSUE THREAD

**Issue by shubha1090** [kyma-project/busola#3146]
1.1 Internal Message Details
Internal Message:	
Test Plan:	ACC SAP Kyma runtime 2024
Message Priority:	High
Affected UI	Button
Affected Check Point:	ACC-264.1 
Short text:	ACC-264.1 (Level A) _ Modify Modules _ Button _ Name of dialog is not read 

1.2 Application Details
Application Name	Kyma-dashboard
Technology:	SAP UI5 
Component:	
URL:	https://dashboard.stage.kyma.cloud.sap/cluster/garden-kyma-stage--c-7074971-external/kymamodules
Username/Password:	NA
1.3 Test Environment Details
Web browser:	Google Chrome Version 127.0.6533.89
Operating System:	Microsoft Windows 11 x64
Screen Reader:	JAWS 2024 Version 2024.2406.121 ILM

1.4 JIRA/SNOW Details
Issue:	Bug
Labels:	Accessibility

2 Screen Flow
•	Launch the application using given URL. 
•	Select the ‘Link to Kyma dashboard’ from the Kyma Environment section on your SAP BTP subaccount page. Now Select ‘Modify Modules’ button which opens ‘Modules’ page.
•	Open JAWS 2024 screen reader and turn on VPC mode.
•	Activate  ‘Sort’ button from the main region of ‘Modules’ page which opens ‘Sort’ dialog.

3. Issue Type: Manual
Observed Behavior:
On activating ‘Sort’ button from the main region of ‘Modules’ page , a new ‘Sort’ dialog is opened  , however the name of the dialog open i.e. ‘Sort’ is not announced by the screen reader instead reads as ‘Modules – Google Chrome dialog’ which confuses the user.
Screenshot:
 

Expected Behavior:
On activating ‘Sort’ button from the main region of ‘Modules’ page , a new ‘Sort’ dialog is opened  , and  the name of the dialog open i.e. ‘Sort’ must be announced by the screen reader along with other necessary information.

Note: 
This issue exists across the application and thus fix should be provided for all the occurrences.
________________________________________________________________________
Kindly refer to attached error and issue processing document for more details.
Regards,
Shubha
